src/app.ts

```diff
@@ -1,7 +1,24 @@
 import "./components/maincontroller";
 
-let senders = cast.framework.CastReceiverContext.getInstance().getSenders();
-let id = senders.length !== 0 && senders[0].id ? senders[0].id : new Date().getTime();
+declare global {
```

Comment:
You can usually create a `d.ts` file for this an put it anywhere in the project.

It'll define the type in the entire project.